app/cdap/components/RecipeManagement/EditRecipe/services.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import MyDataPrepApi from 'api/dataprep';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import { IUpdateRecipeService, IGetRecipeByNameService } from 'components/RecipeManagement/types';
+
+export const getRecipeByNameService = ({
```

Comment:
Can you please check this comment https://github.com/cdapio/cdap-ui/pull/851#discussion_r1071814924 and change accordingly?

Reply:
moved services to common services.ts file inside RecipeManagement Folder

Reply:
moved services to common services.ts file inside RecipeManagement Folder